application/frontend/controllers/SiteController.php

```diff
@@ -2,83 +2,33 @@
 namespace frontend\controllers;
 
 use frontend\components\BaseRestController;
-use yii\filters\AccessControl;
-use yii\helpers\ArrayHelper;
-use yii\web\MethodNotAllowedHttpException;
 use yii\web\ServerErrorHttpException;
-use yii\web\UnauthorizedHttpException;
 
-/**
- * Site controller
- */
 class SiteController extends BaseRestController
 {
-
-    public $layout = false;
-
-    /**
-     * Access Control Filter
-     * REMEMBER: NEEDS TO BE UPDATED FOR EVERY ACTION
-     * @return array
-     */
     public function behaviors()
     {
-        return ArrayHelper::merge(parent::behaviors(), [
-            'access' => [
-                'class' => AccessControl::className(),
-                'rules' => [
-                    [
-                        'allow' => true,
-                        'roles' => ['@'],
-                    ],
-                    [
-                        'allow' => true,
-                        'actions' => ['system-status'],
-                        'roles' => ['?'],
-                    ],
-                ]
-            ],
-            'authenticator' => [
-                'except' => ['system-status'] // bypass authentication for /site/system-status
-            ]
-        ]);
-    }
+        $behaviors = parent::behaviors();
```

Comment:
@fillup Do you have any thoughts on using this approach (get the parent values, make changes, return the result) versus our typical ArrayHelper::merge approach? I think consistency would suggest we keep using merge, but for someone who isn't familiar with the details of how Yii's array merge function works, the get-change-return approach is clearer.